src/frontend/Navigation/AppStack.tsx

```diff
@@ -32,115 +30,175 @@ import { AlreadyOnProj } from "../screens/AlreadyOnProject";
+      <RootStack.Screen name="AddPhoto" component={AddPhoto} />
+      <RootStack.Screen
+        name="AddToProjectScreen"
+        component={AddToProjectScreen}
+      />
+      <RootStack.Screen name="AlreadyOnProj" component={AlreadyOnProj} />
+      <RootStack.Screen name="AppPasscode" component={AppPasscode} />
+      <RootStack.Screen name="BGMapsSettings" component={BGMapsSettings} />
+      <RootStack.Screen name="BackgroundMaps" component={BackgroundMaps} />
+      <RootStack.Screen name="CategoryChooser" component={CategoryChooser} />
+      <RootStack.Screen
+        name="ConfirmLeavePracticeModeScreen"
+        component={ConfirmLeavePracticeModeScreen}
+      />
+      <RootStack.Screen
+        name="ConnectingToDeviceScreen"
+        component={ConnectingToDeviceScreen}
+      />
+      <RootStack.Screen name="CoordinateFormat" component={CoordinateFormat} />
+      <RootStack.Screen
+        name="CreateProjectScreen"
+        component={CreateProjectScreen}
+      />
+      <RootStack.Screen name="DirectionalArrow" component={DirectionalArrow} />
+      <RootStack.Screen name="Experiments" component={Experiments} />
+      <RootStack.Screen name="GpsModal" component={GpsModal} />
+      <RootStack.Screen name="LanguageSettings" component={LanguageSettings} />
+      <RootStack.Screen
+        name="LeaveProjectScreen"
+        component={LeaveProjectScreen}
+      />
+      <RootStack.Screen name="ManualGpsScreen" component={ManualGpsScreen} />
+      <RootStack.Screen name="MapSettings" component={MapSettings} />
+      <RootStack.Screen name="Observation" component={Observation} />
+      <RootStack.Screen
+        name="ObservationDetails"
+        component={ObservationDetails}
+      />
+      <RootStack.Screen name="ObservationEdit" component={ObservationEdit} />
+      <RootStack.Screen name="ObservationList" component={ObservationList} />
+      <RootStack.Screen name="OfflineAreas" component={OfflineAreas} />
+      <RootStack.Screen name="P2pUpgrade" component={P2pUpgrade} />
+      <RootStack.Screen name="PhotosModal" component={PhotosModal} />
+      <RootStack.Screen name="ProjectConfig" component={ProjectConfig} />
+      <RootStack.Screen name="Security" component={Security} />
+      <RootStack.Screen name="Settings" component={Settings} />
+      <RootStack.Screen name="SyncModal" component={SyncModal} />
+      <RootStack.Screen
+        name="UnableToLinkScreen"
+        component={UnableToLinkScreen}
+      />
+    </RootStack.Group>
+    {devExperiments.onboarding && (
+      <RootStack.Group>
+        <OnboardingStack />
+      </RootStack.Group>
+    )}
+    {devExperiments.appPasscode && (
+      <RootStack.Group>
+        <RootStack.Screen name="NewPasscode" component={NewPasscode} />
+        <RootStack.Screen name="AppPasscode" component={AppPasscode} />
+      </RootStack.Group>
+    )}
+    {/* To do: Determine if ICCA */}
+    {false && (
```

Comment:
How are we determining whether the ICCA flag variable is on? This needs to updated to reflect that, but I am not really sure how that's done

Reply:
Right, that makes sense. Happy to turn this back to its own file, but ill wait for gregors input just so we are doing this correctly